packages/utils/src/index.js

```diff
@@ -9,6 +9,8 @@ import {
   Modifier,
 } from 'draft-js'
 import type DraftEntityInstance from 'draft-js/lib/DraftEntityInstance'
+// eslint-disable-next-line
```

Comment:
shouldn't need to use the eslint comment here

Reply:
Eslint will throw error about `deprecated-api`. So i add `eslint-disable-next-line node/no-deprecated-api` in the next commit